go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/kettek/apng v0.0.0-20220622131934-4e92eded13be
 	github.com/lestrrat/go-file-rotatelogs v0.0.0-20180223000712-d3151e2a480f
 	github.com/lib/pq v1.10.6
-	github.com/minio/minio-go/v6 v6.0.57
+	github.com/minio/minio-go/v7 v7.0.47
```

Comment:
likewise for the minio upgrade - if not explicitly required, would be good to see it in a dedicated PR.